VideoPlayer/VideoPlayer.js

```diff
@@ -1429,6 +1433,7 @@ const VideoPlayerBase = class extends Component {
 
 	/**
 	 * Programmatically pauses the current media.
+	 * If pause during fast forwarding or rewinding, the playback speed will be update to normal.
```

Comment:
```suggestion
	 * The playback speed updates to normal if the pause button is pressed during fast forwarding or rewinding.
```

Reply:
or, How about "If you pause during fast forwarding or rewinding, the playback speed updates to normal." ?